compose.yml

```diff
@@ -77,13 +77,14 @@ services:
       KAFKA_LISTENERS: DOCKER://kafka:29092,LOCALHOST://localhost:9092
       KAFKA_ADVERTISED_LISTENERS: DOCKER://kafka:29092,LOCALHOST://localhost:9092
     ports:
-      - ${EXPOSED_KAFKA_PORT:-9092}:9092
+      - ${EXPOSED_KAFKA_PORT:-9092}:29092
+      - ${EXPOSED_KAFKA_PORT2:-29092}:29092
     volumes:
       - ${DATA_DIR:-./data}/kafka:/tmp/logs
 
   init-kafka:
     profiles:
-      - "kafka-init"
+      - "kafka"
```

Comment:
if we remove the `kafka-init` profile, then please remove it also from the README
tho, kafka initialization will run every run.